kairon/api/app/routers/bot/customer_orders.py

```
@@ -0,0 +1,166 @@
+    CreateOrderRequest,
+    UpdateOrderStatusRequest,
+    FilterOrdersRequest,
+)
+from kairon.shared.models import User
+
+router = APIRouter()
+
+@router.post("/customer", response_model=Response)
+async def upsert_customer(
+        request_data: UpsertCustomerRequest,
+        current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS),
```

Ruff (0.16.1)

[warning] Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)